app/src/components/TimestampOrderBookScroller.tsx

```diff
@@ -68,36 +70,48 @@ class TimestampOrderBookScroller extends Component<Props> {
      * @param e
      */
     onKeyUp = e => {
-        if (e.keyCode === LEFT_ARROW_KEY_CODE) this.handleGoToPreviousMessage();
-        else if (e.keyCode === RIGHT_ARROW_KEY_CODE) this.handleGoToNextMessage();
+        const { playback } = this.props;
+        if (!playback) {
+            if (e.keyCode === LEFT_ARROW_KEY_CODE) this.handleGoToPreviousMessage();
+            else if (e.keyCode === RIGHT_ARROW_KEY_CODE) this.handleGoToNextMessage();
+        }
     };
 
     /**
      * @desc handler for the event of scrolling to the top of the book entity
      */
     handleScrollToTopOfTheBook = () => {
-        this.middleReferenceItem
-        && this.middleReferenceItem.current
-        && this.middleReferenceItem.current.scrollIntoView({
-            behavior: 'smooth',
-            block: 'center',
-        });
+        const { playback } = this.props;
+        if (!playback) {
+            this.middleReferenceItem
+            && this.middleReferenceItem.current
+            && this.middleReferenceItem.current.scrollIntoView({
+                behavior: 'smooth',
+                block: 'center',
+            });
+        }
```

Comment:
I think the button should be visually disabled if this feature is not accessible

Reply:
ok, will make it disabled.